easybuild/easyconfigs/m/MFEM/MFEM-4.7-foss-2023a.eb

```
@@ -0,0 +1,50 @@
+easyblock = 'CMakeMake'
+
+name = 'MFEM'
+version = '4.7'
+
+homepage = 'https://mfem.org/'
+description = "MFEM is a general, modular library for finite element methods."
+
+toolchain = {'name': 'foss', 'version': '2023a'}
+
+source_urls = ['https://github.com/mfem/releases/raw/gh-pages/']
+sources = ['mfem-4.7.tgz']
+checksums = ['5e889493f5f79848f7b2d16afaae307c59880ac2a7ff2315551c60ca54717751']
+
+toolchainopts = {'usempi': True, 'openmp': True}
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+]
+
+dependencies = [
+    ('Hypre', '2.29.0'),
+    ('METIS', '5.1.0'),
+    ('OpenBLAS', '0.3.23'),
+    ('OpenMPI', '4.1.5'),
+    ('SuiteSparse', '7.1.0'),
+]
+
+configopts = ' '.join([
+    '-DMFEM_USE_MPI=YES',
+    '-DMFEM_USE_METIS_5=YES',
+    '-DMFEM_USE_SUITESPARSE=YES',
+    '-DMFEM_USE_NETCFD=NO,',
```

```suggestion
    '-DMFEM_USE_NETCDF=NO,',
```

was being ignored due to a typo

```
-- Generating done (0.4s)
CMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_INSTALL_LIBDIR
    MFEM_USE_NETCFD


-- Build files have been written to: /home/crivella/.local/easybuild/build/MFEM/4.7/foss-2023a/easybuild_obj
```